fastlane_core/spec/configuration_spec.rb

```diff
@@ -134,6 +134,46 @@
         end
       end
 
+      describe "#sensitive flag" do
+        it "should set the sensitive flag" do
+          config_item = FastlaneCore::ConfigItem.new(key: :foo,
+                                                     description: 'foo',
+                                                     type: Array,
+                                                     optional: true,
+                                                     sensitive: true,
+                                                     default_value: ['5', '4', '3', '2', '1'])
+          expect(config_item.sensitive).to eq(true)
+        end
+        it "should ask using asterisks", now: true do
+          config_item = FastlaneCore::ConfigItem.new(key: :foo,
+                                                     description: 'foo',
+                                                     type: String,
+                                                     is_string: true,
+                                                     optional: false,
+                                                     sensitive: true)
+          config = FastlaneCore::Configuration.create([config_item], {})
+          allow(FastlaneCore::Helper).to receive(:is_test?).and_return(false)
+          allow(FastlaneCore::Helper).to receive(:interactive?).and_return(true)
+          allow(FastlaneCore::Helper).to receive(:ci?).and_return(false)
```

Comment:
configuration.rb -> `fetch`  disables itself, if `is_test?` - non `interactive?`  or `ci?`
to test if plain or password is used, i mock them - hope this is ok.